### DISCUSSION TITLE
Is it possible to show the TOC both in body and right?

### DISCUSSION THREAD

**Discussion by AlFontal** [quarto-dev/quarto-cli#3336] [Q&A]
As far as I know, it's possible to select a single location to show the Table of Contents, but while the dynamic TOC that always stays on the right is useful, sometimes I also want to show a full TOC just after the header in an article/blog post. 

Is there any mechanism to have it behave like that right now?

**Comment by dragonstyle** [comment]
You can control the location of the TOC using `toc-location` specifying either `left`, `right`, or `body` (which I think is what you're looking for). A little more on TOC customization here:

https://quarto.org/docs/output-formats/html-basics.html#table-of-contents


HTH!

**Comment by AlFontal** [comment]
Hi @dragonstyle,

Thanks. I am aware of the `toc-location` parameter, but my question was in regards to being able to use _two_ different locations at the same time for the `toc-location`.  

Of course, having both `left` and `right` TOC doesn't make sense, but I would like something akin to the combination of `left` and `body` or `right` and body:

For instance, I usually always want the dynamic right side TOC, but I would like to start a post/report with a full TOC written in the main body. I don't think any of the options for `toc-location`, or at least nothing that I could find in the documentation, allows for that.

Is there maybe a similar method to what is used for references, that would instruct Quarto to print a full TOC at a certain div? 

This seems to work for references:

```
::: {#refs}
:::
```

Could we/do we have something like this:

```
::: {#toc}
:::
```
Thanks!

**Reply by dragonstyle** [reply]
Ahh I see- this definitely isn't possible right now (as you suspected). I'll convert this thread to an issue to track adding support for this improvement!

**Reply by AlFontal** [reply]
Great thanks!

**Comment by ghost** [comment]
Moved to #3473
